### PR TITLE
Twig cleanup

### PR DESCRIPTION
**Pull request by CarsonF**

### CODE REVIEW

**Comment by GawainLynch** [comment]
Epic!

File: src/Storage/Entity/ContentValuesTrait.php [hunk 1]
```diff
@@ -646,9 +646,13 @@ protected function getTemplateFieldsContentType()
...
-            $template = $this->app['templatechooser']->record($this);
-            if (array_key_exists($template, $templateFieldsConfig)) {
-                return $templateFieldsConfig[$template];
+            $templateChooser = $this->app['templatechooser'];
+            /** @var \Bolt\TemplateChooser $template */
+            $templates = $templateChooser->record($this);
+            foreach ($templates as $template) {
+                if (array_key_exists($template, $templateFieldsConfig)) {
+                    return $templateFieldsConfig[$template];
+                }
```
Comment (CarsonF) [inline src/Storage/Entity/ContentValuesTrait.php]: Should this use twig to resolve the template like I did with `TemplateFieldsType`?

Comment (GawainLynch) [inline src/Storage/Entity/ContentValuesTrait.php]: Would be nice.

Comment (CarsonF) [inline src/Storage/Entity/ContentValuesTrait.php]: But....?

Comment (GawainLynch) [inline src/Storage/Entity/ContentValuesTrait.php]: No buts … 
make-it-so

Comment (CarsonF) [inline src/Storage/Entity/ContentValuesTrait.php]: #6237

**Pull request merged**